.github/workflows/release.yml

```diff
@@ -16,23 +25,44 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '16'
+          cache: 'yarn'
       - name: Setup Git
         run: |
           git config user.name "GitHub Bot"
           git config user.email "gituser@example.com"
-      - uses: actions/setup-node@v3
-        with:
-          node-version: '14'
-          cache: 'yarn'
       - name: Install packages
         run: yarn install --frozen-lockfile
+      - name: Lint
```

Comment:
Why do we need lint when releasing?

Reply:
Just an additional layer of check, Making sure, it doesn't break lints... 

The `pre-release` workflow can be skipped for PR by adding `[skip CI]` to the commit message.

This check can make sure, it still runs the build even when it `pre-release` workflow is skipped.

Reply:
im not sure we can benefit from lint stage if the code already merged to main
but, if you think its relevant, keep it :)